addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -235,6 +235,18 @@ export class ProductScreen extends ControlButtonsMixin(Component) {
         }
     }
     async onClickPay() {
+        let is_paid = await this.rpc({
```

Comment:
use `const` rather than `let` since the variable isn't being changed
Also use a try/catch for case where connection was lost. We don't wanna block the user if he's offline
And this should only be run if we are sharing orders (so if the order has a `server_id`) otherwise it's pointless rpc for people not sharing